openmc/executor.py

```diff
@@ -44,7 +56,7 @@ def plot_geometry(output=True, openmc_exec='openmc', cwd='.'):
 
 
 def run(particles=None, threads=None, geometry_debug=False,
-        restart_file=None, tracks=False, output=True, cwd='.',
+        restart_file=None, tracks=False, output="full", cwd='.',
```

Comment:
Use single quotes around `'full'`, just to be consistent with the convention across the rest of the API.